enhancements/network-observability.md

```diff
@@ -0,0 +1,367 @@
+a control plane perspective instead of device by device.  In addition,
+it can assist in the areas of network planning, network policy validation,
+security, and others.
+
+### Goals
+
+- Help troubleshoot network-related issues at the control plane level
+- Show what is happening on the network in a single cluster
+- Gather insight and be able to answer questions about the network
+- Provide rich visualization at the network layer, irrespective of the
+underlying layer
+
+The goal of the first release is to lay the groundwork and foundation
+in place, while still being able to deliver some functionality, even
+if it is at a smaller scale.  The target is to have a Dev Preview to
+generate interest in network observability.
+
+### Non-Goals
+
+The focus is on network observability as opposed to other types of observability.
+
+While a key goal is troubleshooting, it is not strictly a troubleshooting
+tool.  Some of that are already addressed by
+[Red Hat Insights](https://cloud.redhat.com/blog/openshift-insights-for-openshift-cluster-manager)
+and specific tools like pcap, traceroute, and [Jaeger](https://www.jaegertracing.io/)
+for tracing.
+
+
+## Proposal
+
+Network Observability is an opt-in feature that needs to be enabled
+by a user with an admin or cluster-admin role.  This is done by installing
+the Network Observability Operator, which in turn, installs the dependency
+operators necessary for this feature.  The user can do this using the
+web console or the CLI.
+
+In the initial release, Open vSwitch (OVS) will be configured to export
+IPFIX data.  The data will be collected and combined with Kubernetes-related
+information (e.g. pod, services, namespaces) and then saved in internal or
+external storage.
+
+The web console will provide a NetFlow table showing traffic between
+pods.  In the future, more visualization and functionality will be
+provided including areas such as topology, network-related eBPF data,
+policy validation, security risks, and more.
+
+***Note:*** *The term "NetFlow" is used generically throughout this document and
+is synonymous with IPFIX, which is the IETF version of NetFlow.*
+
+### User Stories
+
+1. As a cluster/network admin, I need access to data that allows me to debug network problems and conduct audits of network traffic.
+    - “I’m trying to target traffic towards a specific pod, and the traffic coming into the cluster isn’t getting there.”
+    - “The traffic coming into the cluster isn’t landing where I expect, where do things go wrong?”
+    - “I’m trying to understand why traffic isn’t flowing from pod to pod in the way that I expected.”
+
+2. As a cluster/network admin, I need basic search and filtering tools to analyze a subset of NetFlow data inside the OpenShift console.
+    - Users can filter NetFlow based on traffic source or destination (ie, view all of the traffic associated with a particular node, service, namespace, pod, etc.).
```

Comment:
It seems the proposal addresses user story 2, but not 1.  The first seems more level and aspirational for work over a longer period of time.  I'd clarify this somehow.